src/audio_sdl_mixer.cpp

```diff
@@ -242,8 +242,12 @@ void SdlMixerAudio::BGM_OnPlayedOnce() {
 
 	if (!bgm_stop) {
 		played_once = true;
-		// Play indefinitely without fade-in
+
+// introduced in SDL2_mixer 2.0.2, which deprecated libmad
+#ifndef SDL_MIXER_COMPILEDVERSION
 		if (Mix_GetMusicType(bgm.get()) != MUS_MP3_MAD)
+#endif
+			// Play indefinitely without fade-in
```

Comment:
Multiline if without brackets.

Reply:
Imo the whole MUS_MP3_MAD if-check can be removed because for the next cleanup all the other ugly hacks for <= 2.0.1 will be removed, too.

Reply:
I am fine with this. 👍